src/SymfonyEventDispatcherAdapter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+namespace Bnf\SymfonyPsrEventDispatcherAdapter;
```

Comment:
could you use the right namespace "TYPO3" here, please? Thanks

Reply:
Ahhhh, sorry.. messed that up…
Sure..

Reply:
Fixed, to be `TYPO3\SymfonyPsrEventDispatcher`, but now the class name is ``TYPO3\SymfonyPsrEventDispatcher\SymfonyEventDispatcherAdapter`.
So pretty much a repition, was once with "Psr", and once without :\